Source/Core/DolphinQt2/Config/GeckoWidget.h

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QWidget>
```

Comment:
`<string>`, `<vector>`, and `"Common/CommonTypes.h"` need to be included.

Reply:
Done.